docs/source/federation/implementing.md

```diff
@@ -125,7 +125,11 @@ const gateway = new ApolloGateway({
 })();
 ```
 
-In this example, we provide the `serviceList` option to the `ApolloGateway` constructor, which provides a name and endpoint for each of the federated services. The name (an arbitrary string) is primarily used for query planner output, error messages, and logging.
+In this example, we provide the `serviceList` configuration to the `ApolloGateway` constructor, which provides a name and endpoint for each of the federated services. The name (an arbitrary string) is primarily used for query planner output, error messages, and logging.
+
+It's important to ensure that **federated services are not exposed publicly**. Because of the power and flexibility of the `_entities` field, it may present a serious risk to expose it. The model of federation implies that the gateway implements the entirety of the graph and uses the federated services beneath it in its implementation, so there should never be a need for a client to reach these services directly. If, for some reason, you want to allow outside traffic to send operations to implementing services, we highly recommend keeping the `_entities` field restricted.
```

Comment:
> If, for some reason, you want to allow outside traffic to send operations to implementing services, we highly recommend keeping the `_entities` field restricted.

What would be the suggestion for implementing this restriction?   I think a more simple security model here would be sticking to the suggestion that federated services not be exposed publicly.